pandas/core/indexes/numeric.py

```diff
@@ -403,7 +403,7 @@ def __contains__(self, other):
     @Appender(_index_shared_docs['get_loc'])
     def get_loc(self, key, method=None, tolerance=None):
         try:
-            if np.all(np.isnan(key)):
+            if np.all(np.isnan(key)) or isinstance(key, bool):
```

Comment:
use is_bool here

Reply:
Does python have `is_bool()` function?
I used `isinstance()` to check if key is an instance of subclass of bool.

Reply:
Oh sorry, I found it.

Reply:
pandas has all of these functions, which we consistently use. thanks